.commitlintrc.js

```diff
@@ -9,6 +9,6 @@ module.exports = {
       'always',
       ['all', 'deps', 'deps-dev', 'packages', 'release', 'root', ...readdirSync('./packages')],
     ],
-    'scope-empty': [2, 'never'],
+    'scope-empty': [1, 'never'],
```

Comment:
This just changes the lint error into a warning right?

Reply:
Yes